repositories/OptionRepository.js

```diff
@@ -19,24 +32,41 @@ module.exports.findAll = function findAll(
     .then(map(fromDb));
 };
 
+module.exports.findOne = function findOne(
```

Comment:
This should be named `getById` to follow the convention of other repositories.

Reply:
This isn't quite the same functionality of getById, as I needed a function where I could pass it a number of different conditions and have it return only the first result. I suppose the best option here would be to specialise it further and name it getExclusiveOption. Plus we already have a getById in here.

Reply:
Sorry.  Mis-read the function. This is identical to `findAll()` apart from the `head()` right at the end.  I would either utilise the `findAll()` method to prevent duplication or leave the resposibility of `head()` to the caller.

Reply:
Not sure if you saw the change but I ended up specialising the function so it only relates to Exclusives now, makes it a bit easier to call as only the answerId needs passing to it plus it makes the function itself simpler.